app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -499,27 +485,39 @@ fun <T> SettingCard(
                     .align(Alignment.End)
                     .clickable {
                         if (isEnabled) {
-                            dialogOpen = true
+                            if (!isPermissions) {
```

Comment:
Maybe rename from `isPermissions` to `onEditShowDialog` or something similar, more abstract and there will be no need to rename it in the future, if another setting appears that requires the dialog to be not opened and is not associated with permissions anyhow. Or just make `editContent` nullable (wrap into `()?`) and check if it is null, if it is, means we have nothing to show in the dialog, so we don't.